Assets/Scripts/Models/FurnitureJobs.cs

```diff
@@ -0,0 +1,214 @@
+    /// </summary>
+    /// <param name="furn">The current furniture.</param>
+    /// <param name="other">The prototype furniture.</param>
+    public FurnitureJobs(Furniture furn, Furniture other)
+    {
+        furniture = furn;
+        activeJobs = new List<Job>();
+        pausedJobs = new List<Job>();
+
+        SpawnSpotOffset = other.Jobs.SpawnSpotOffset;
+        WorkSpotOffset = other.Jobs.WorkSpotOffset;
+    }
+
+    /// <summary>
+    /// Gets the spot offset where the Character will stand when he is using the furniture. This is relative to the bottom
+    /// left tile of the sprite. This can be outside of the actual furniture.
+    /// </summary>
+    /// <value>The spot offset where the Character will stand when he uses the furniture.</value>
+    public Vector2 WorkSpotOffset { get; private set; }
+
+    /// <summary>
+    /// Gets the spot offset where inventory is spawn when a Job is done with this machine.
+    /// </summary>
+    /// <value>The spawn spot offset.</value>
+    public Vector2 SpawnSpotOffset { get; private set; }
+
+    /// <summary>
+    /// Gets the <see cref="Job"/> with the specified index.
+    /// </summary>
+    /// <param name="i">The index.</param>
+    public Job this[int i]
+    {
+        get
+        {
+            return activeJobs[i];
+        }
+    }
+
+    /// <summary>
+    /// Gets the tile that is used to do a job.
+    /// </summary>
+    /// <returns>Tile that is used for jobs.</returns>
+    public Tile GetWorkSpotTile()
+    {
+        return World.Current.GetTileAt(furniture.Tile.X + (int)WorkSpotOffset.x, furniture.Tile.Y + (int)WorkSpotOffset.y, furniture.Tile.Z);
+    }
+
+    /// <summary>
+    /// Gets the tile that is used to spawn new objects (i.e. Inventory, Character).
+    /// </summary>
+    /// <returns>Tile that is used to spawn objects (i.e. Inventory, Character).</returns>
+    public Tile GetSpawnSpotTile()
+    {
+        return World.Current.GetTileAt(furniture.Tile.X + (int)SpawnSpotOffset.x, furniture.Tile.Y + (int)SpawnSpotOffset.y, furniture.Tile.Z);
+    }
+
+    /// <summary>
+    /// Reads the job work spot offset from the xml.
+    /// </summary>
+    /// <param name="reader">The Xml Reader.</param>
+    public void ReadWorkSpotOffset(XmlReader reader)
+    {
+        WorkSpotOffset = new Vector2(
+            int.Parse(reader.GetAttribute("X")),
+            int.Parse(reader.GetAttribute("Y")));
+    }
+
+    /// <summary>
+    /// Reads the spawn spot offset from the xml.
+    /// </summary>
+    /// <param name="reader">The Xml Reader.</param>
+    public void ReadSpawnSpotOffset(XmlReader reader)
+    {
+        SpawnSpotOffset = new Vector2(
+            int.Parse(reader.GetAttribute("X")),
+            int.Parse(reader.GetAttribute("Y")));
+    }
+
+    /// <summary>
+    /// How many active jobs are linked to this furniture.
+    /// </summary>
+    /// <returns>The number of jobs linked to this furniture.</returns>
+    public int Count()
```

Comment:
I was kind of going with the idea that you didn't need to know about the implementation of how active and paused jobs worked, and was going for similar names as before. Since all paused jobs are canceled jobs, and we are storing them in a way similar to holding a prototype, Count returns the amount of jobs linked to this furniture. In a way you don't Add a paused or active job, you add a job and we deal with it. I can change it for ActiveCount (since you always call it as Jobs.ActiveCount).

I just change the lists names internally to make it clear.